R/db-odbc-oracle.R

```diff
@@ -45,6 +45,15 @@ sql_translate_env.Oracle <- function(con) {
   )
 }
 
+#' @export
+db_explain.Oracle <- function(con, sql, ...) {
+  exsql <- build_sql("EXPLAIN PLAN FOR ", sql, con = con)
+  invisible(dbGetQuery(con, exsql))
```

Comment:
What's happening here? Why do you ignore the output of this query?

Can we use `DBI::dbExecute()` instead?

Reply:
Yes, you're right, that would be more straightforward